### DISCUSSION TITLE
Feature request: Rename by a slow double click and Create a new folder auto escaping special characters

### DISCUSSION THREAD

**Discussion by iwillneverstop** [doublecmd/doublecmd#1137] [Suggestions]
Thank you for a nice File Manager.

I'm using Double Command v0.8.2 on both Windows and Linux.

I have simple feature requests:

1. When I give a slow double click on a file or folder, Double Command allows me to rename the file or folder. Double Command should select the whole folder name or the whole file name (without file extension) so that I can start typing a new name right away. This feature is available in Windows' File Explorer.

2. When I create a new folder (pressing F7) and give a name containing special characters (e.g. ':'), Double Command can automatically escape those special characters (e.g. by removing them or replacing them with white spaces) so that I don't need to manually find and delete those special characters to have a valid name. This feature is available in SpeedCommander on Windows.

Thank you in advance.

**Comment by Mannshoch** [comment]
May If you use a `/` or `\` inside the name (e.g. 'This is/a name.txt') and press enter, ask if this file should be moved and rename the file into a subdirectory 'this is' and name it 'a name.txt'

**Comment by avmaksimov** [comment]
1. It already exists now.
2. I don't think it's needed feature because there will be big chance to miss the mistake.
